walkthrough/utils.py

```diff
@@ -134,3 +136,107 @@ def generate_step(cookiecutter_path: Path, data: Dict[str, Any]) -> Path:
         )
 
     return DATASET_DIR
+
+
+def _check_env() -> None:
+    """Check if environment variables are set correctly."""
+    po.put_markdown("""## Checking `.env` file...""")
+
+    ok = True
+    for env_name in ("GRAPHER_USER_ID", "DB_USER", "DB_NAME", "DB_HOST"):
+        if getattr(config, env_name) is None:
+            ok = False
+            po.put_warning(
+                po.put_markdown(f"Environment variable `{env_name}` not found, do you have it in your `.env` file?")
+            )
+
+    if ok:
+        po.put_success(po.put_markdown("`.env` configured correctly"))
+
+
+def _show_environment() -> None:
+    """Show environment variables."""
+    po.put_markdown("## Environment")
+    po.put_info(
+        po.put_markdown(
+            f"""
+    * **GRAPHER_USER_ID**: `{config.GRAPHER_USER_ID}`
+    * **DB_USER**: `{config.DB_USER}`
+    * **DB_NAME**: `{config.DB_NAME}`
+    * **DB_HOST**: `{config.DB_HOST}`
+    """
+        )
+    )
+    if OWIDEnv().env_type_id == OWIDEnvType.LIVE:
+        po.put_warning(
+            po.put_markdown(
+                "You are currently connected to the **live** database! If you are not sure what this means, please"
+                " consult the with ETL team."
+            )
+        )
+
+
+class OWIDEnvType(Enum):
+    LIVE = 1
+    STAGING = 2
+    LOCAL = 3
+    UNKNOWN = -1
```

Comment:
It's not clear that using an enum with integer codes is better than strings for this type.

Would it be simpler if it was just a literal?

```python
OWIDEnvType = Literal["live", "staging", "local", "unknown"]
```

That would shorten a bunch of code, and it would still get type-checked properly.